.ci/build-windows.yml

```diff
@@ -33,7 +33,7 @@ steps:
       customCommand: 'run build'
 
   - task: Npm@1
-    displayName: 'npm run test:e2e'
+    displayName: 'jest --runInBand test-e2e'
```

Comment:
Would be nice to try without `--runInBand` sometime, maybe it will be faster, test suite itself is designed to be run concurrenctly.